com.unity.render-pipelines.high-definition/Runtime/Material/Fullscreen/HDFullscreenFunctions.hlsl

```diff
@@ -0,0 +1,6 @@
+
+// For now we fake the GI sampling in fullscreen.
+float3 SampleBakedGI(float3 positionRWS, float3 normalWS, float2 uvStaticLightmap, float2 uvDynamicLightmap)
```

Comment:
maybe better to not expose functions that dont have functionality?

Reply:
Yeah, I've marked the "BakedGINode" in ShaderGraph incompatible with the fullscreen target